<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>517</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1087</width>
    <height>517</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1087</width>
    <height>517</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MagSearch</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/logo/logo.ico</normaloff>:/logo/logo.ico</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="input_lineedit">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>15</y>
      <width>715</width>
      <height>20</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="search_pushbutton">
    <property name="geometry">
     <rect>
      <x>738</x>
      <y>15</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="result_treewidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>54</y>
      <width>802</width>
      <height>415</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="indentation">
     <number>10</number>
    </property>
    <property name="expandsOnDoubleClick">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="headerCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>265</number>
    </attribute>
    <attribute name="headerHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="headerShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>名称</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>大小</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>链接</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="filter_lineedit">
    <property name="geometry">
     <rect>
      <x>831</x>
      <y>54</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="use_filter_checkbox">
    <property name="geometry">
     <rect>
      <x>1002</x>
      <y>54</y>
      <width>70</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>名称包含</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="less_than_lineedit">
    <property name="geometry">
     <rect>
      <x>945</x>
      <y>87</y>
      <width>34</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="more_than_lineedit">
    <property name="geometry">
     <rect>
      <x>861</x>
      <y>87</y>
      <width>34</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>831</x>
      <y>87</y>
      <width>28</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>大于:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>915</x>
      <y>87</y>
      <width>28</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>小于:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="size_limit_checkbox">
    <property name="geometry">
     <rect>
      <x>1002</x>
      <y>87</y>
      <width>67</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>大小限制</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="source_treewidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>828</x>
      <y>153</y>
      <width>240</width>
      <height>313</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="headerDefaultSectionSize">
     <number>50</number>
    </attribute>
    <column>
     <property name="text">
      <string>启用</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>名称</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>828</x>
      <y>132</y>
      <width>130</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>来源选项:(待添加功能)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>897</x>
      <y>87</y>
      <width>16</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>MB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>981</x>
      <y>87</y>
      <width>16</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>MB</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="action">
   <property name="text">
    <string>配置源</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>导入配置</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
